src/app/kbv/controllers/prove-identity-another-way.js

```diff
@@ -12,6 +12,28 @@ class ProveIdentityAnotherWayController extends BaseController {
       callback(err, locals);
     });
   }
+
+  async saveValues(req, res, next) {
+    await super.saveValues(req, res, async (err) => {
+      if (err) {
+        return next(err);
+      }
+      const choice = req.body["prove-identity-another-way"];
+      const history = req.journeyModel.get("history");
+
+      if (choice === "stop") {
+        return res.redirect("/oauth2/callback?error=access_denied");
+      }
+
+      let redirect = "/kbv/answer-security-questions";
+      if (history && history.length > 0) {
+        const last = history[history.length - 1];
+        redirect = last.next;
+      }
+
+      return res.redirect(redirect);
+    });
```

Comment:
Can we handle more of this using `next` in `steps.js` rather than doing this routing logic in here `saveValues`? see how it's handled for Experian KBV: https://github.com/govuk-one-login/ipv-cri-kbv-front/blob/fb3df9f97960330a3a783fd0e96e5be4a0e0b70f/src/app/kbv/steps.js#L29